core/domain/state_domain.py

```diff
@@ -1586,11 +1586,11 @@ def update_interaction_answer_groups(self, answer_groups_list):
         self._update_content_ids_in_assets(
             old_content_id_list, new_content_id_list)
 
-    def update_interaction_default_outcome(self, default_outcome_dict):
+    def update_interaction_default_outcome(self, default_outcome):
         """Update the default_outcome of InteractionInstance domain object.
 
         Args:
-            default_outcome_dict: dict. Dict that represents Outcome domain
+            default_outcome: object. object that represents Outcome domain
```

Comment:
`<parameter>: <type> <explanation>`, Please add the class name of the object in the palce of `<type>` (Outcome?)

Reply:
changed